identity-enabler/deviceId-mobile-app/src/lib/ui/helpers.ts

```diff
@@ -53,3 +55,18 @@ export function multiClick(node: HTMLElement, opts?: { numClicks?: number; maxDe
 
     node.addEventListener("click", handleClick);
 }
+
+/**
+ * Wraps fly transition to ensure element is on top during outro or intro.
+ *
+ * @param node
+ * @param opts
+ */
+export function flyOver(node: HTMLElement, opts: FlyParams) {
+    const z = +getComputedStyle(node).zIndex;
```

Comment:
performance-wise getComputedStyle is very costly

Reply:
In this case it's only called once prior to animation, I think it's safe usage.